src/sentry/snuba/spans_rpc.py

```diff
@@ -135,21 +137,30 @@ def run_timeseries_query(
     params: SnubaParams,
     query_string: str,
     y_axes: list[str],
-    groupby: list[str],
-) -> Any:
-    pass
+    referrer: str,
+    granularity_secs: int,
+    config: SearchResolverConfig,
+) -> SnubaTSResult:
     """Make the query"""
-    # maker = SearchResolver(params)
-    # groupby, contexts = maker.resolve_columns(groupby)
-    # yaxes = maker.resolve_aggregate(y_axes)
-    # query = maker.resolve_query(query_string)
+    rpc_request = get_timeseries_query(
+        params, query_string, y_axes, [], referrer, config, granularity_secs
+    )
 
     """Run the query"""
-    # rpc = timeseries_RPC(columns=[column.proto_definition for column in groupby], query=query)
-    # result = rpc.run()
+    rpc_response = snuba_rpc.timeseries_rpc(rpc_request)
 
     """Process the results"""
-    # return _process_timeseries(result, columns)
+    result: SnubaData = []
```

Comment:
What about the `_process_timeseries` function defined below?

Reply:
Oh woops, forgot about that, i'll move this over